examples/mautic-example-nginx-ssl-managedDB/README.md

```diff
@@ -0,0 +1,22 @@
+This example shows how to configure nginx to act as a reverse proxy with ssl support for mautic container.
+
+#### What is handled in this docker-compose.yml:
+
+* Setup of mysql and mautic containers
+* Setup of mysql db on managed or remote server
+* Setup of a nginx container with custom vhost configuration (nginx.conf)and secure with letsencrypt certs
+
+#### How to use this example:
+
+1. run ```docker-compose up``` in this directory
```

Comment:
I think that we only require one backtick for an inline comment but if there is a reason to use three please feel free to disregard the suggestion!

```suggestion
1. Run `docker-compose up` in this directory
```